README.md

```diff
@@ -171,6 +315,7 @@ Talaiot will send to the InfluxDb server defined in the configuration the values
 | publishBuildMetrics          | Publish build metrics of the publisher, true by default                             |
 | publishTaskMetrics           | Publish tasks metrics of the publisher, true by default                             |
 
+Included in: `com.cdsap.talaiot` and `com.cdsap.talaiot.plugin.influxdb` plugins
```

Comment:
Nit: all "Included in" sections in the document miss a period.
```suggestion
Included in: `com.cdsap.talaiot` and `com.cdsap.talaiot.plugin.influxdb` plugins.
```

Reply:
done